src/components/Dashboard/TodoList.vue

```diff
@@ -902,7 +897,8 @@ export default class TodoList extends Vue {
     const header = filing.header
 
     if (dissolution && business && header) {
-      if (!this.isGoodStanding && this.isStatusDraft(header)) {
+      // don't allow resuming a draft if not in good standing
+      if (this.isStatusDraft(header) && !this.isGoodStanding) {
```

Comment:
I switched the conditions as it's easier to read this way.

Note that logic was probably copy-pasted from loadAlteration() and I think it's incorrect here and in loadSpecialResolution() below, but that's outside the scope of this ticket.